jsonpath4k/build.gradle.kts

```diff
@@ -43,8 +61,9 @@ val generateKotlinGrammarSource = tasks.register<AntlrKotlinTask>("generateKotli
     arguments = listOf("-visitor")
 
     // Generated files are output inside build/generatedAntlr/{package-name}
-    val outDir = "generatedAntlr/${packageName!!.replace(".", "/")}"
-    outputDirectory = layout.buildDirectory.dir(outDir).get().asFile
+    val outDir = "$SRCDIR_ANTRL/${packageName!!.replace(".", "/")}"
+    outputDirectory = layout.projectDirectory.dir(outDir).asFile
+
```

Comment:
sure, but why? Is that more in line with the default source set layout?

Reply:
I concede! I just noticed the files are built on gradle sync, not when executing a task.
Moving them to the build folder would actually be painful whenever the clean task is executed, as this would require a new gradle sync each time

Please consider this issue resolved and the pull request approved